### ISSUE TITLE
Remove references to Zend_Measure_Weight

### ISSUE THREAD

**Issue by sanderjongsma** [netresearch/dhl-shipping-m2#59]
Hi there, Can you please remove all references to Zend_Measure_Weight and replace with `vendor/magento/framework/Measure/Weight.php`. Currently Magento is forcing out all Zend related classes and is migrating to Laminas. But in you case  all weight references can be found in the Weight class of magento. 

If this is fixed the composer requirement to `magento/zendframework1` can be removed in the composer.json also. Magento also doesn't use this requirement anymore, only for dev usage. 

The reason for this issue is that your module now gives a lot of warnings during composer install like this: 

```
Warning: Ambiguous class resolution, "Zend_Cache_Backend" was found in both "/data/plutosport/magento2/vendor/magento/zend-cache/library/Zend/Cache/Backend.php" and "/data/plutosport/magento2/vendor/magento/zendframework1/library/Zend/Cache/Backend.php", the first will be used.
Warning: Ambiguous class resolution, "Zend_Cache_Manager" was found in both "/data/plutosport/magento2/vendor/magento/zend-cache/library/Zend/Cache/Manager.php" and "/data/plutosport/magento2/vendor/magento/zendframework1/library/Zend/Cache/Manager.php", the first will be used.
Warning: Ambiguous class resolution, "Zend_Db" was found in both "/data/plutosport/magento2/vendor/magento/zend-db/library/Zend/Db.php" and "/data/plutosport/magento2/vendor/magento/zendframework1/library/Zend/Db.php", the first will be used.

```

**Comment by mam08ixo** [comment]
After diving a little deeper into the topic, the situation is as follows:

The `Zend_*` classes that we use throughout the packages were moved to `magento/framework` and various `magento/zend-*` packages in 2.4.6. While the latter can also be installed on 2.4.5 (leading to the same warning message when generating autoload files there), the former is simply not available on 2.4.5.

That means that, as long as we support M2.4.5, the requested change will not be made. The earliest date for us to drop 2.4.5 support is general availability of 2.4.7 – the latest date is end of software support for 2.4.5 ([Adobe Commerce lifecycle policy](https://experienceleague.adobe.com/docs/commerce-operations/release/planning/lifecycle-policy.html)).

Since the warning message is an annoyance but does not break things, we hope for your understanding. Thank you for bringing this topic to our attention.

**Comment by Sebastian80** [comment]
Hi @sanderjongsma, 

This was done with Release 2.11.0
